### PR TITLE
[Admin] Non Lethal Smiting

### PR DESCRIPTION
**Pull request by JamieD1**
Pies

#### Changelog

:cl:  
rscadd: Pies
/:cl:

### CODE REVIEW

**Comment by MegaEmpirical** [comment]
Now make it explode!

**Comment by ToGWtF** [comment]
speedmerge

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1147,6 +1147,12 @@ Traitors and the like can also be revived with the previous role mostly intact.
 			if(!puzzle_imprison(target))
 				to_chat(usr,"<span class='warning'>Imprisonment failed!</span>", confidential=TRUE)
 				return
+		if(ADMIN_PUNISHMENT_PIE)
+			var/mob/living/carbon/C = target
+			var/confirm = alert(usr, "Send honk message?", "Honk Message", "Yes", "No")
+			if(confirm == "Yes")
```
Comment (Hopekz) [inline code/modules/admin/verbs/randomverbs.dm]:
```suggestion
			var/confirm = alert(usr, "Send honk message?", "Honk Message", "Yes", "No")
			if(confirm == "Yes")
			var/mob/living/carbon/C = target
```
Why:
If you quit the game before answering your text box that target variable will always be set.
Does it matter in the long run? No, but it looks prettier

**Review by Hopekz — CHANGES_REQUESTED** [review]
Otherwise ready for merge

**Comment by JamieD1** [comment]
Hows that

**Review by Hopekz — APPROVED** [review]

**Pull request merged**